pyart/util/__init__.py

```diff
@@ -42,5 +42,6 @@
 from .circular_stats import mean_of_two_angles, mean_of_two_angles_deg
 from .xsect import cross_section_ppi
 from .hildebrand_sekhon import estimate_noise_hs74
+from .radar_utils import is_vpt, to_vpt
```

Comment:
These functions should be added to the Miscellaneous function list in the module docstring so that these function are included in the documentation.